man/machinery-inspect.1.md

```diff
@@ -68,6 +68,10 @@ trigger errors.
 
       $ `machinery` inspect --skip-files=/file\\,with_comma myhost
 
+  * `--verbose` (optional):
+    Display the filters which are used during inspection at the beginning.
```

Comment:
I find the `at the beginning` a bit confusing. Can we just remove it here and in the man page?